config/packages/flysystem.yaml

```diff
@@ -0,0 +1,6 @@
+flysystem:
+    storages:
+        sylius.storage:
+            adapter: 'local'
+            options:
+                directory: '%sylius_core.images_dir%'
```

Comment:
```suggestion
```
as it is already defined in `src/Sylius/Bundle/CoreBundle/Resources/config/app/config.yml`